tutorial/practicals/nemo/4_nemo_openacc/solutions/kernels_trans.py

```diff
@@ -63,22 +63,16 @@
 ACC_KERNELS_TRANS = ACCKernelsTrans()
 
 
-def trans(psy):
+def trans(psyir):
     '''A PSyclone-script compliant transformation function.
```

Comment:
Not yours but "...function that applies OpenACC KERNELS around suitable loops."

Reply:
Done

Reply:
done